g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/translation/dao/impl/LocalSingleComponentDaoImpl.java

```diff
@@ -124,6 +124,7 @@ public boolean writeTranslationToFile(ComponentMessagesDTO componentMessagesDTO)
 			LOGGER.info("The bunlde file path {}{} is found, update the bundle file.", basePath, filepath );
 			try {
 				SortJSONUtils.writeJSONObjectToJSONFile(basePath + filepath, componentMessagesDTO);
+				LOGGER.info("The bunlde file {}{} have updated sucessfully", basePath, filepath);
```

Comment:
Should add separator between basePath and filepath to read easily?

Reply:
There doesn't need to add the filepath. Because of the base path must end of filepath in configuration